<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="label_text">
        <property name="text">
         <string>测试信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_text">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Microsoft New Tai Lue</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>选择你的网卡</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>过滤规则</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="tableView"/>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="data">
           <property name="font">
            <font>
             <family>SimSun-ExtB</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>开始</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>结束</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="23" margin="14"/>
 <resources/>
 <connections/>
</ui>
